cmd/crc/cmd/daemon_windows.go

```diff
@@ -17,5 +19,14 @@ func vsockListener() (net.Listener, error) {
 }
 
 func httpListener() (net.Listener, error) {
-	return nil, nil
+	ln, err := winio.ListenPipe(constants.DaemonHTTPNamedPipe, &winio.PipeConfig{
+		MessageMode:      true,  // Use message mode so that CloseWrite() is supported
+		InputBufferSize:  65536, // Use 64KB buffers to improve performance
+		OutputBufferSize: 65536,
+	})
+	logging.Infof("listening %s", constants.DaemonHTTPNamedPipe)
```

Comment:
dunno if it should be `Info` or `Debug`?

Reply:
I found it useful to see something from the daemon when starting it.